x11-misc/mate-notification-daemon/mate-notification-daemon-1.27.1.ebuild

```diff
@@ -21,13 +19,13 @@ SLOT="0"
 IUSE="nls X wayland"
 REQUIRED_USE="|| ( X wayland )"
 
-COMMON_DEPEND="dev-libs/atk
+COMMON_DEPEND="
+	>=app-accessibility/at-spi2-core-2.46.0
 	>=dev-libs/glib-2.50:2
 	>=dev-libs/libxml2-2.9.0
 	>=sys-apps/dbus-1
 	x11-libs/cairo
 	>=x11-libs/gdk-pixbuf-2.22:2
-	>=x11-libs/libnotify-0.7
```

Comment:
What's this bit about?

Reply:
`$REQ_GLIB_VERSION gio-2.0 >= $REQ_GLIB_VERSION gtk+-3.0 >= $GTK_REQUIRED libnotify" 2>/dev/null``

From what I understand this is still required because of this but please correct me if I'm wrong.

Reply:
oh wait I see what you mean